pkg/issuer/acme/dns/util/wait_test.go

```diff
@@ -75,7 +75,7 @@ var checkAuthoritativeNssTestsErr = []struct {
 	error       string
 }{
 	// invalid nameserver
-	{"8.8.8.8.asn.routeviews.org.", "fe01=", []string{"invalidns.com."},
+	{"8.8.8.8.asn.routeviews.org.", "fe01=", []string{"invalid.example.com."},
```

Comment:
Afaik, `invalidns.com` isn't rfc reserved, so it's a bad choice for testing purposes.

Also, I can't tell if it means `invalid ns` or `invalid dns`. I'm assuming that it doesn't matter, but if one or the other is preferred, i'd recommend `invalid-dns.example.com` or `invalid-ns.example.com`.